### DISCUSSION TITLE
Ukraine language pack

### DISCUSSION THREAD

**Discussion by pekarskyi** [YOURLS/YOURLS#3543] [Show and tell]
uk (Ukraine) language pack is done, based on 1.9.2

Hi, I am Mykola Pekarskyi. I am WordPress Developer and translator from Ukraine.

YOURLS uk GitHub repository: [here](https://github.com/Dizer7/YOURLS-uk)

I will maintain this language pack for a long time. It's not difficult for me.

I will be glad if you add the link of this Ukrainian localization to your official list of translations. Thank you!

_Many thanks for the script YOURLS. He is wonderful!_

**Comment by dgw** [comment]
Hey! Thanks for putting this together. You can open a pull request yourself to list this translation at https://github.com/YOURLS/awesome#translations

**Reply by pekarskyi** [reply]
Done!

**Reply by dgw** [reply]
x-ref: YOURLS/awesome#138
